BraintreeDropIn/Public/BTCardFormViewController.h

```diff
@@ -43,7 +43,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, strong, nullable, readonly) BTCardCapabilities *cardCapabilities;
 
 /// The card network types supported by this merchant
-@property (nonatomic, copy) NSArray *supportedCardTypes;
+@property (nonatomic, copy) NSArray<NSNumber *> *supportedCardTypes;
```

Comment:
Can we use `<BTUIKPaymentOptionType *>` with the enum name here to make this more readable?

Reply:
Unfortunately, we can't. In Objective-C, NSArrays can only hold NSObjects, and `BTUIKPaymentOptionType` is an enum, not an NSObject.